gui/wxpython/gis_set.py

```diff
@@ -524,7 +524,7 @@ def SuggestDatabase(self):
             mapset_name = create_startup_location_in_grassdb(grassdb,
                                                              location)
             if mapset_name:
-                self.SetLocation(grassdb, location, mapset_name)
+                self.SetLocation(grassdb, location, "PERMANENT")
```

Comment:
This would defeat the idea of promoting proper use of mapsets (subprojects). Note that you don't have to be in the mapset to add a map from it, although in this case, you may want to consider showing the PERMANENT mapset subtree in data catalog.